mars-sim-core/src/main/java/org/mars_sim/msp/core/tool/SimulationConstants.java

```diff
@@ -0,0 +1,15 @@
+package org.mars_sim.msp.core.tool;
+
+import org.mars_sim.msp.core.Simulation;
+import org.mars_sim.msp.core.UnitManager;
+import org.mars_sim.msp.core.environment.SurfaceFeatures;
+import org.mars_sim.msp.core.person.ai.mission.MissionManager;
+
+public interface SimulationConstants {
+
+  static Simulation sim = Simulation.instance();
```

Comment:
In the future I want to reduce the dependency on ```Simulation.instance``` as it can cause problem when a simulation is loaded.
The simulation code generally relies too much on static references but there is no need to change it yet.